pkg/broker/broker.go

```diff
@@ -94,7 +166,11 @@ func (b *Broker) Provision(request *osb.ProvisionRequest, _ *broker.RequestConte
 
 	klog.V(4).Infof("broker: provisioning request %+v in namespace %q", request, namespace)
 
-	operationName, err := b.Client.Provision(request.InstanceID, request.ServiceID, request.PlanID, namespace, request.AcceptsIncomplete, request.Parameters)
+	params, found := b.overrideChartParams.ForService(request.ServiceID)
```

Comment:
Maybe stick a shiny comment here explaining this behaviour? This is the most important part of the feature.